zscript/StupidAchievements.zs

```diff
@@ -379,8 +390,11 @@ class sa_ // namespace
                       , int borderX, int borderY
                       , int borderWidth, int borderHeight
                       , int boxWidth, int boxHeight
+                      , int iconX, int iconY
+                      , int iconWidth, int iconHeight
                       , double alpha
                       , TextureID texture
+					   , TextureID Icon
```

Comment:
Here and in other places: please use spaces only, and preserve formatting.

Reply:
These alignments are completely my mistakes. I fixed.